manim/scene/graph_scene.py

```diff
@@ -101,7 +101,7 @@ def setup_axes(self, animate=False):
             color=self.axes_color,
             include_tip=self.include_tip,
             add_start=self.x_add_start,
-            add_end=self.x_add_end
+            add_end=self.x_add_end,
```

Comment:
It seems like black wants these commas.
I don't like them too much, but I can live with it.

Reply:
Embrace the way of black! ;)